Sources/Logging/LogHandler.swift

```diff
@@ -0,0 +1,152 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Logging API open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the Swift Logging API project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift Logging API project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+/// A `LogHandler` is an implementation of a logging backend.
+///
+/// This type is an implementation detail and should not normally be used, unless implementing your own logging backend.
+/// For user API of please refer to the documentation of `Logger`.
+///
+/// # Implementation requirements
+///
+/// To implement your own `LogHandler` you should respect a few requirements that are necessary so applications work
+/// as expected irregardless of the `LogHandler` implementation chosen by the application.
+///
+/// - make your `LogHandler` a `struct`
+/// - changing the metadata or the `logLevel` on one `Logger` must not change any `Logger`-typed variable's log levels.
```

Comment:
This sentence confused me. How about:

• The metadata and `logLevel` properties must be implemented so that setting them on a `Logger` does not affect other `Logger`s